app/lib/submission_builder/state1099_g.rb

```diff
@@ -25,21 +25,23 @@ def document
                       form1099g.intake.spouse
                     end
         xml.RecipientSSN recipient.ssn
-        xml.RecipientName sanitize_for_xml(recipient.full_name)
+        xml.RecipientName sanitize_for_xml(recipient.full_name, 35)
         xml.RecipientUSAddress do
           xml.AddressLine1Txt sanitize_for_xml(form1099g.recipient_address_line1, 35)
           xml.AddressLine2Txt sanitize_for_xml(form1099g.recipient_address_line2, 35) if form1099g.recipient_address_line2.present?
           xml.CityNm sanitize_for_xml(form1099g.recipient_city, 22)
           xml.StateAbbreviationCd state_abbreviation
           xml.ZIPCd sanitize_for_xml(form1099g.recipient_zip)
         end
-        xml.UnemploymentCompensation form1099g.unemployment_compensation_amount&.round
-        xml.FederalTaxWithheld form1099g.federal_income_tax_withheld_amount&.round
-        xml.State1099GStateLocalTaxGrp do
-          xml.StateTaxWithheldAmt form1099g.state_income_tax_withheld_amount&.round
-          xml.StateAbbreviationCd state_abbreviation
-          if form1099g.state_identification_number && form1099g.state_identification_number != ''
-            xml.PayerStateIdNumber form1099g.state_identification_number
+        xml.UnemploymentCompensation form1099g.unemployment_compensation_amount&.round if form1099g.unemployment_compensation_amount.present?
+        xml.FederalTaxWithheld form1099g.federal_income_tax_withheld_amount&.round if form1099g.federal_income_tax_withheld_amount.present?
+        if form1099g.state_income_tax_withheld_amount.present?
```

Comment:
so it looks like `StateAbbreviationCd` is the only required node within the `State1099GStateLocalTaxGrp`, so maybe we don't need to check if the `state_income_tax_withheld_amount` is present here?

Reply:
I tried this locally -- 
we cannot leave state_income_tax_withheld_amount field blank (we get a validation error)
however, we _can_ enter `0` -- not sure if this is intentional or if we should add a validation error to not allow it? (Q for product)